center/src/main/java/com/microsoft/hydralab/center/service/AgentManageService.java

```diff
@@ -128,4 +130,39 @@ public void updateAgentTeam(String teamId, String teamName){
 
          agentUserRepository.saveAll(agents);
     }
+
+    public File downloadAgentConfigFile(SysUser requestor, String agentId) {
```

Comment:
1. The requestor has been checked in the controller, please remove the SysUser parameter.
2. BTW, the method name could be **generateAgentConfigFile**